### PR TITLE
[tests-only][full-ci]resourcetype variable deleted from testcode

### PR DESCRIPTION
**Pull request by S-Panta**
This PR deletes the step implementation of test code along with its related steps regarding resource type variable.
Related issue: https://github.com/owncloud/ocis/issues/8349

### CODE REVIEW

**Review by PrajwolAmatya — COMMENTED** [review]
lgtm

File: tests/acceptance/features/apiSharingNg/deletePermissions.feature [hunk 1]
```diff
@@ -92,34 +89,32 @@ Feature: Remove access to a drive item
...
   Scenario Outline: user removes access to a folder in link share
     Given user "Alice" has created folder "FolderToShare"
     And user "Alice" has created the following link share:
-      | resourceType    | folder        |
       | resource        | FolderToShare |
       | space           | Personal      |
       | permissionsRole | <role>        |
       | password        | %public%      |
-    When user "Alice" removes the share permission of link from folder "FolderToShare" of space "Personal" using the Graph API
+    When user "Alice" removes the share permission of link from "FolderToShare" of space "Personal" using the Graph API
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/deletePermissions.feature:117]:
```suggestion
    When user "Alice" removes the share permission of link from folder "FolderToShare" of space "Personal" using the Graph API
```

File: tests/acceptance/features/bootstrap/SharingNgContext.php [hunk 2]
```diff
@@ -470,12 +456,11 @@ public function removeSharePermission(
 	}
 
 	/**
-	 * @When /^user "([^"]*)" removes the share permission of (user|group) "([^"]*)" from (file|folder) "([^"]*)" of space "([^"]*)" using the Graph API$/
+	 * @When /^user "([^"]*)" removes the share permission of (user|group) "([^"]*)" from "([^"]*)" of space "([^"]*)" using the Graph API$/
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/SharingNgContext.php:459]:
```suggestion
	 * @When /^user "([^"]*)" removes the share permission of (user|group) "([^"]*)" from (?:file|folder) "([^"]*)" of space "([^"]*)" using the Graph API$/
```

File: tests/acceptance/features/bootstrap/SharingNgContext.php [hunk 3]
```diff
@@ -487,20 +472,18 @@ public function userRemovesSharePermissionOfUserFromResourceOfSpaceUsingGraphAPI
...
 	): void {
 		$this->featureContext->setResponse(
-			$this->removeSharePermission($sharer, $shareType, $resourceType, $resource, $space)
+			$this->removeSharePermission($sharer, $shareType, $resource, $space)
 		);
 	}
 
 	/**
-	 * @When /^user "([^"]*)" removes the share permission of link from (file|folder) "([^"]*)" of space "([^"]*)" using the Graph API$/
+	 * @When /^user "([^"]*)" removes the share permission of link from "([^"]*)" of space "([^"]*)" using the Graph API$/
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/SharingNgContext.php:484]:
```suggestion
	 * @When /^user "([^"]*)" removes the share permission of link from (?:file|folder) "([^"]*)" of space "([^"]*)" using the Graph API$/
```

**Review by saw-jan — CHANGES_REQUESTED** [review]
we should keep this

**Review by saw-jan — APPROVED** [review]

**Pull request merged**